help/de/Help_ch_Account-Actions.xml

```diff
@@ -89,337 +103,350 @@
+<!-- <footnote><para>Previous versions of this help defined A/P and A/R the other way round.</para></footnote> -->
             </entry>
           </row>
 
-          <row>
-            <entry namest="c1">
-              <para>Kreditkarte
+          <row rowsep="1">
+            <entry namest="c2" nameend="c3">
+              <para>(Anlagevermögen; langfristig)
               </para>
             </entry>
+          </row>
 
+          <row>
             <entry namest="c2">
-              <para>Die Kontoart Kreditkarte kommt für die Konten zu zum Einsatz die, im Gegensatz zu normalen
-                Bankkonten, häufig im Minus sind. Diese Kontoart kann sowohl für Kreditkarten die
-                eine gleitende Kreditlinie erlauben (z. B. VISA, oder MasterCard) als auch bei
-                Karten, die keine laufenden Guthaben zulassen (z. B. American Express), verwendet
-                werden.
+              <para>Aktienkonto
               </para>
             </entry>
-          </row>
 
-          <row>
-            <entry namest="c1">
-              <para>Devisenhandel
+            <entry namest="c3">
+              <para>Die Aktie (englisch share) ist ein Wertpapier, das den Anteil an einer Aktiengesellschaft oder einer
+                Kommanditgesellschaft auf Aktien (KGaA) verbrieft.
+                <footnote>
+                  <para><ulink url="&url-wp-de;Aktie"/>
+                  </para>
+                </footnote>
+
+                <footnote>
+                  <para><tip>
+                      <para>Solange Kryptowährungen von &app; nicht als Währungen unterstützt werden, wird empfohlen, sie
+                        ebenfalls als Aktie anzulegen.
+                      </para>
+                    </tip>
+                  </para>
+                </footnote>
               </para>
             </entry>
+          </row>
 
+          <row rowsep="1">
             <entry namest="c2">
-              <para>Währungskonten wurden für den Handel mit Devisen verwendet, sind aber durch den Bereich
-                "Währungstransfer" im Fenster "Geldtransfer" ersetzt worden
-                (<xref linkend="trans-win-enter"></xref>).
+              <para>Investmentfonds
+              </para>
+            </entry>
+
+            <entry namest="c3">
+              <para>Ein Portfolio aus Aktien, Anleihen oder anderen Geldanlagen, das sich in Anteile teilen läßt.
               </para>
             </entry>
           </row>
 
-          <row>
-            <entry namest="c1">
-              <para>Eigenkapital
+          <row rowsep="1">
+            <entry namest="c1" morerows="4">
+              <para>Passiva
               </para>
             </entry>
-<!-- FIXME source: insert link and <menuchoice> -->
+
+            <entry namest="c2" nameend="c3">
+              <para>Passiva bezeichnet die Mittelherkunft und unfasst das auf der rechten Bilanzseite verzeichnete
+                Eigen- und Fremdkapital eines Betriebes.
```

Comment:
`Passiva stehen auf der rechten Seite der Bilanz und be`{zechnen|schreiben}` die Mittelherkunft. Sie bestehen aus dem Eigen- und Fremdkapital Ihres <quote>Betriebes</quote>.`

Reply:
ok.